git/gitutil.go

```diff
@@ -64,10 +64,22 @@ func (g *Util) SplitPath() (string, string, error) {
 	return strings.Trim(root, "\n"), strings.Trim(sub, "\n"), nil
 }
 
-// CloneRepository clones a local repository to a new location on disk.
-func CloneRepository(src, dst string) error {
-	_, err := runGitCmd("/", "clone", src, dst)
-	return err
+// CloneRepository clones a repository to a new location on disk. If paths are
```

Comment:
If no paths are provided, nothing is being checked out. I find this a bit misleading as I would expect the entire things to be checked out if no paths are provided. Should we change the wording or the behaviour below?

Reply:
If no paths are provided, then it does a normal clone of everything.

Reply:
Sorry, somehow missed that `if`!